.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ubuntu-20.04, macos-latest, windows-latest]
```

Comment:
Why fix this? Does it not work on latest?

Reply:
Got it. Might be worth dropping 3.6 at some point.

Reply:
if dropping 3.6 gets us latests versions, something to consider in future

Reply:
It shows that as of 2021-12-23, 3.6 has already reached the [end of life phase](https://peps.python.org/pep-0494/#lifespan). So it doesn't seem to be supported any more in ubuntu releases. I think it makes sense to drop 3.6.